stoobly_agent/app/cli/endpoint_cli.py

```diff
@@ -54,3 +55,28 @@ def apply(**kwargs):
 
   facade = EndpointFacade(settings)
   facade.apply(context)
+
+@endpoint.command(
+  "import",
+  help="Import endpoints"
+)
+@click.option('--source-format', required=True, type=click.Choice([OPENAPI_FORMAT]), help="Spec file format.")
+@click.option('--source-path', help='Path to spec file.')
+def import_endpoint(**kwargs):
+  context = EndpointsImportContext()
+
+  project_key = settings.proxy.intercept.project_key
+  if project_key:
+    project_key = validate_project_key(project_key)
+    context.with_project(project_key.id)
+  
+  context.with_source(kwargs.get('source_path'), kwargs.get('source_format'))
+
+  endpoint_handler = lambda endpoint: print(f"{bcolors.OKBLUE}Importing Endpoint: {endpoint['method']} {endpoint['path']}{bcolors.ENDC}")
+  context.with_endpoint_handler(endpoint_handler)
+
+  error_handler = lambda endpoint: print(f"{bcolors.FAIL}Failed to Import Endpoint: {endpoint['method']} {endpoint['path']} - Aborting Operation{bcolors.ENDC}")
```

Comment:
Since we are outputting an error, we may want to print to `stderr` instead of `stdout`